cmd/virt-handler/kubevirt_container.cil

```diff
@@ -0,0 +1,10 @@
+(block kubevirt_container
```

Comment:
Just wondering. Did you change the name to `kubevirt_container` because the `container` policy already exists?
I left this one to match the `container` definition in [1]
Perhaps we can somehow import it from [1] at some point... 

[1] https://github.com/containers/udica/blob/master/udica/templates/base_container.cil

Reply:
yes, exactly. "container" comes from udica, so we would be unable to install the RPM on systems with that.

Reply:
oh, ok :)